editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -865,7 +894,11 @@ export const NavigatorRowLabel = React.memo((props: NavigatorRowLabelProps) => {
         paddingLeft: 10,
         paddingRight: props.codeItemType === 'map' ? 0 : 10,
         backgroundColor:
-          isCodeItem && !props.selected ? colorTheme.dynamicBlue10.value : 'transparent',
+          isCodeItem && !props.selected
+            ? colorTheme.dynamicBlue10.value
+            : isOutlet && !props.selected
+            ? colorTheme.aqua10.value
+            : 'transparent',
```

Comment:
this expression should be factored out into a separate function for readability (the same way the callback of `remixItemType` is implemented)